influx_interface.py

```diff
@@ -9,14 +11,24 @@
 # For development only
 import sys, traceback
 
+is_version2 = db_settings['version'] == 2
```

Comment:
This will raise a `KeyError` if the user has not uncommented the new `db_settings` in config.py, lines 28-34.

Perhaps check for the existence of the key first with `if 'version' in db_settings.keys()`?

Reply:
I like it!